locale/hu.json

```diff
@@ -31,256 +31,256 @@
+                    "label": "Teleportálás a célponthoz",
+                    "error": "Nincs célpont kijelölve."
                 },
                 "coords": {
-                    "title": "Coords",
-                    "label": "Go to specified coords",
-                    "dialog_title": "Teleport",
-                    "dialog_desc": "Provide coordinates in an x, y, z format to go through the wormhole.",
-                    "dialog_error": "Invalid coordinates. Must be in the format of: 111, 222, 33"
+                    "title": "Koordináták",
+                    "label": "Menj egy speciális koordinátára",
+                    "dialog_title": "Teleportálás",
+                    "dialog_desc": "A koordinátát x, y, z formában kell megadnod.",
+                    "dialog_error": "Hibás koordináta. Ilyen formát használj: 111, 222, 33"
                 },
                 "back": {
-                    "title": "Back",
-                    "label": "Go back to last location",
-                    "error": "You don't have a last location to go back to!"
+                    "title": "Vissza",
+                    "label": "Vissza az utolsó pozíciódra",
+                    "error": "Nincs hová visszamenned!"
                 },
                 "copy": {
-                    "title": "Copy Coords",
-                    "label": "Copy coords to clipboard."
+                    "title": "Koordináta másolása",
+                    "label": "Jelenlegi koordináta vágólapra másolása."
                 }
             },
             "vehicle": {
-                "title": "Vehicle",
-                "onesync_error": "This option requires that OneSync be on for server entity handling",
-                "not_in_veh_error": "You are not currently in a vehicle!",
+                "title": "Jármű",
+                "onesync_error": "Ez a beállítás használatához a OneSync-nek engedélyezve kell lennie",
+                "not_in_veh_error": "Jelenleg nem ülsz járműben!",
                 "spawn": {
-                    "title": "Spawn",
-                    "label": "Spawn vehicle by model name",
-                    "dialog_title": "Spawn vehicle",
-                    "dialog_desc": "Enter in the model name of the vehicle you want to spawn.",
-                    "dialog_success": "Vehicle spawned!",
-                    "dialog_error": "The vehicle model name '%{modelName}' does not exist!",
-                    "dialog_info": "Trying to spawn %{modelName}."
+                    "title": "Lehívás",
+                    "label": "Jármű lehívása modell név alapján",
+                    "dialog_title": "Jármű lehívása",
+                    "dialog_desc": "Add meg a modell nevét amit le szeretnél hívni.",
+                    "dialog_success": "Jármű lehívva!",
+                    "dialog_error": "Jármű ezzel a modell névvel '%{modelName}' nem létezik!",
+                    "dialog_info": "Megpróbálás lehívni %{modelName}."
```

Comment:
These could use some tweaking, we should discuss.

Reply:
> These could use some tweaking, we should discuss.

For me, this was the best fit, but I’m happy to get ideas on how it could be better